src/recordlinker/linking/mpi_service.py

```diff
@@ -107,3 +107,25 @@ def insert_blocking_keys(
     if commit:
         session.commit()
     return values
+
+def get_all_algorithm_labels(session: orm.Session) -> list[str]:
```

Comment:
I'm thinking we should have a separate service module for these functions.  The intention of this module was just to interact with the `mpi_*` tables.

Reply:
I can move these to a new file. Any ideas for the name? Something like `algorithm_service.py`, `config_service.py`, etc?

Reply:
algorithm_service.py works for me